src/axom/sidre/spio/IOManager.cpp

```diff
@@ -216,7 +216,15 @@ void IOManager::write(sidre::Group* datagroup,
     }
     SLIC_ASSERT(h5_file_id >= 0);
 
-    std::string group_name = fmt::sprintf("datagroup_%07d", m_my_rank);
+    std::string group_name;
```

Comment:
Style question: Why not initialize `group_name = "datagroup"` like the other changes?

Reply:
I agree that it should be initialized on construction, if possible. @nselliott is there a reasonable default?

Reply:
Yes, this is an inconsistency that I missed.  I will address it.